### PR TITLE
feat(sdk): add missing fs.appendFile 

### PR DESCRIPTION
**Pull request by CJLA**
Fixes https://github.com/winglang/wing/issues/5392

## Checklist

- [x] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [x] Description explains motivation and solution
- [x] Tests added (always)
- [x] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Review by eladb — APPROVED** [review]
Great job! 👏

File: libs/wingsdk/src/fs/fs.ts [hunk 1]
```diff
@@ -364,6 +364,21 @@ export class Util {
     fs.writeFileSync(filepath, contents.join("---\n"));
   }
 
+  /**
+   * Appends new data to the end of an existing file
+   * @param filepath The file path that needs to be written.
```
Comment (hasanaburayyan) [inline libs/wingsdk/src/fs/fs.ts:369]:
```suggestion
   * @param filepath The file path that needs to be appended.
```

**Review by hasanaburayyan — CHANGES_REQUESTED** [review]
@CJLA Great work! Can you please add some unit tests for this? The file you will want to do that in is `libs/wingsdk/test/fs/fs.test.ts` Should be some examples there, once you add that Ill take another look and should be good to ship 👍

**Comment by CJLA** [comment]
> @CJLA Great work! Can you please add some unit tests for this? The file you will want to do that in is `libs/wingsdk/test/fs/fs.test.ts` Should be some examples there, once you add that Ill take another look and should be good to ship 👍

@hasanaburayyan I'm trying to run the unit tests in `libs/wingsdk/test/fs/fs.test.ts` and get this error.

`Error: Cannot find module '../package.json'
Require stack:
- /Users/christopherjohnson/wing/libs/wingsdk/test/fs/target/test/fs.test.ts.wsim/.wing/ts/src/constants.js
- /Users/christopherjohnson/wing/libs/wingsdk/test/fs/target/test/fs.test.ts.wsim/.wing/ts/src/core/app.js
- /Users/christopherjohnson/wing/libs/wingsdk/test/fs/target/test/fs.test.ts.wsim/.wing/ts/src/core/index.js
- /Users/christopherjohnson/wing/libs/wingsdk/test/fs/target/test/fs.test.ts.wsim/.wing/ts/src/fs/fs.js
- /Users/christopherjohnson/wing/libs/wingsdk/test/fs/target/test/fs.test.ts.wsim/.wing/ts/test/fs/fs.test.js
- /Users/christopherjohnson/wing/[worker eval]`

I think it's looking for `libs/wingsdk/package.json` which I do see in my files. I was able to successfully run the other unit tests I wrote in `examples/tests/sdk_tests/fs/basic.test.w`

**Comment by hasanaburayyan** [comment]
@CJLA can you confirm what command your using to test, as well as your node version?

**Comment by CJLA** [comment]
@hasanaburayyan 

`pnpm wing test libs/wingsdk/test/fs/fs.test.ts`
Node v20.11.1

**Comment by hasanaburayyan** [comment]
@CJLA hmmm what happens if you cd into `libs/wingsdk` and just run `npx vitest fs`

**Comment by CJLA** [comment]
@hasanaburayyan that worked :thumbsup:

**Review by hasanaburayyan — APPROVED** [review]
@CJLA Great job, thanks so much for sticking it through to get the unit tests in. As well CONGRATS on your first contribution! 🔥 🎉 Looking forward to the next!

**Pull request merged**